README.md

```diff
@@ -175,75 +186,44 @@ to change s
 
 ### Arrays
 
-**Definition:** This variable is used to have inside more variales even it self
-
-*Example:*  v_list = [4.4, 16, v_list2=['hello'], false]
-
-Lists start indexing from number 0.
-
-Indexation
-
-can access to this index with
```

Comment:
these are guides to know the way of coding

Reply:
I don't get why or what ur saying here